charts/external-dns/templates/deployment.yaml

```
@@ -4,6 +4,10 @@ metadata:
   name: {{ include "external-dns.fullname" . }}
   labels:
     {{- include "external-dns.labels" . | nindent 4 }}
+{{- with .Values.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
```

```suggestion
    {{- toYaml . | nindent 4 }}
```

This chart favours `nindent` for this pattern.

---

Thanks for your suggestions. I have changed to `deploymentAnnotations` and now using `nindent`